src/søknad/steg/5-aktivitet/Firma/OmFirmaetDitt.tsx

```diff
@@ -127,26 +129,46 @@ const OmFirmaetDitt: React.FC<Props> = ({
       </FeltGruppe>
 
       {firma.navn?.verdi && (
-        <FeltGruppe>
-          <Input
-            label={labelOrganisasjonsnr}
-            bredde={'L'}
-            type={'text'}
-            onChange={(e) =>
-              settInputTekstFelt(
-                e,
-                EFirma.organisasjonsnummer,
-                labelOrganisasjonsnr
-              )
-            }
-            value={
-              firma?.organisasjonsnummer ? firma?.organisasjonsnummer.verdi : ''
-            }
-          />
-        </FeltGruppe>
+        <>
+          <FeltGruppe>
+            <Input
+              label={labelOrganisasjonsnr}
+              bredde={'L'}
+              type={'text'}
+              onChange={(e) =>
+                settInputTekstFelt(
+                  e,
+                  EFirma.organisasjonsnummer,
+                  labelOrganisasjonsnr
+                )
+              }
+              value={
+                firma?.organisasjonsnummer
+                  ? firma?.organisasjonsnummer.verdi
+                  : ''
+              }
+              feil={
+                firma.organisasjonsnummer?.verdi &&
+                !erStrengGyldigOrganisasjonsnummer(
+                  firma?.organisasjonsnummer?.verdi
```

Comment:
Denne kan kanskje trekkes ut som en konstant ettersom den brukes to ganger? 
```const harValgtUgyldigOrganisasjonsnummer =                 firma.organisasjonsnummer?.verdi &&
                !erStrengGyldigOrganisasjonsnummer(
                  firma?.organisasjonsnummer?.verdi)
```

Reply:
Det kan den! :) Fikser